src/server.go

```diff
@@ -184,7 +183,7 @@ func (a *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
       return
     }
 
-    if a.protect && rec.deleted == NO {
+    if !unlink && a.protect && rec.deleted == NO {
```

Comment:
Wow, did UNLINK just not work? We should add a test.

Reply:
yeah tests are MIA for unlink and protect

Reply:
+1